### DISCUSSION TITLE
Can I set a display name for a foreign key

### DISCUSSION THREAD

**Discussion by kmasong22** [pocketbase/pocketbase#1658] [Q&A] [answered]
Hello,

I am very new to pocketbase and I am wondering if it is possible to show a display name using the fields in a collections instead of the foreign key?

Also, is it possible to specify which field will be used to select in a relation type field. Currently, from what I noticed it shows the first field in alphabetical order when selecting from the dropdown.

**Comment by ganigeorgiev** [comment, marked as answer]
No, this is not possible at the moment.

The relation picker and listing UI is in the process of being refactored per #976 and its related issues; the improvements will be available with the next v0.12.0 release soon.

**Reply by kmasong22** [reply]
Thank you for responding immediately and thank you for the information.

May I ask if this option could potentially be available in future releases of pocketbase?

**Reply by ganigeorgiev** [reply]
> May I ask if this option could potentially be available in future releases of pocketbase?

Yes, it will be available in the next v0.12.0 release soon. If you haven't already, you can check the roadmap and current tasks priority in https://github.com/orgs/pocketbase/projects/2.

**Reply by kmasong22** [reply]
Thank you, I just reviewed the roadmap. Thank you for making pocketbase.
